llm-example/llm.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+import requests
+
+SYSTEM_PROMPT = os.getenv("SYSTEM_PROMPT") or ""
+
+
+def llm(prompt):
+    response = requests.post(
+        os.getenv("TRUEFOUNDRY_API_ENDPOINT"),
```

Comment:
we should raise error if this key is not present.

on the top of this file do:

TFY_LLM_GATEWAY_HOST = os.environ["TFY_LLM_GATEWAY_HOST"] 
TFY_API_KEY = os.environ["TFY_API_KEY"]

also rename the env vars (the way I have commented)